src/utils/logger/logger.config.ts

```diff
@@ -0,0 +1,23 @@
+const isDevelopment = process.env.NODE_ENV === 'development';
+import type { LoggerOptions } from 'pino';
+
+const LOGGER_CONFIG: LoggerOptions = {
+  level: isDevelopment ? 'debug' : 'info',
+  formatters: {
+    level(label, number) {
+      return { level: isDevelopment ? number : label };
+    },
+  },
+  browser: {
+    transmit: {
+      level: 'warn',
+      send: (level, info) => {
+        // testing send
```

Comment:
WIP - I'll add a Pino HTTP transport here so that we print logs from web in the server's stdout.